src/kontrol/kompile.py

```diff
@@ -121,6 +124,7 @@ def foundry_kompile(
             imports=_imports,
             keccak_lemmas=options.keccak_lemmas,
             auxiliary_lemmas=options.auxiliary_lemmas,
+            evm_rule_optimizations=options.evm_rule_optimizations,
```

Comment:
Let's also set `evm_rule_optimizations` to `true` in `apply_optimizations(self)`. This will include the optimization in the `--optimize-performance N` flag.

Reply:
Nevermind, this is a flag for build and `optimize-performance` is a flag for `prove`.